### ISSUE TITLE
Some doubts regarding fine-tuning 

### ISSUE THREAD

**Issue by Henry-Ellis** [thu-ml/RoboticsDiffusionTransformer#20]
Thank you for your excellent contribution. I have fine-tuned the model of my mechanical arm with 170m and 1b, but the results are not satisfactory and the success rate is not high. I use single-arm data and two camera views, head and wrist, based on your suggestion, I only use right_arm, left_arm=0 and the image of the left wrist camera is the same as the image of the right wrist camera.I only have one task using a single instruction. I want to know where my problem lies and whether the fine-tuning process is reasonable. I look forward to your reply. 

The following images are my inference visualization
<img src="https://github.com/user-attachments/assets/36610efa-01ea-4386-afb0-3c5e716145a8" width="400px">

**Comment by csuastt** [comment]
>  the image of the left wrist camera is the same as the image of the right wrist camera

Do not use the same image... Please PAD the left-wrist camera.

**Comment by csuastt** [comment]
Yeah, your result is quite abnormal. I speculate that the training is not converged. Maybe you should train longer.

Besides, I have a few questions:
1. How many episodes are used?
2. What is the total batch size?
3. May I take a look at your loss curve?
4. Any other bugs?

**Comment by Henry-Ellis** [comment]
> Yeah, your result is quite abnormal. I speculate that the training is not converged. Maybe you should train longer.
> 
> Besides, I have a few questions:
> 
> 1. How many episodes are used?
> 2. What is the total batch size?
> 3. May I take a look at your loss curve?
> 4. Any other bugs?


**Thank you for your reply, i will trt PAD later. Did you try the single-arm data when you fine-tuned it？** 
1.About 1000 episodes single task
2. --train_batch_size=4 \ --sample_batch_size=8 \
3.loss doesn't seem to be a problem、
4.Not yet
<img src="https://github.com/user-attachments/assets/5ded9de2-ece1-4568-9bf9-6222f6ee046b" width="400px">

Very happy to follow your advice.

**Comment by csuastt** [comment]
> Thank you for your reply, i will trt PAD later. Did you try the single-arm data when you fine-tuned it？

No, I didn't. But I don't think that is the point.

How many GPUs are used? I got that batch size per GPU is 4.

**Comment by Henry-Ellis** [comment]
> How many GPUs are used? I got that batch size per GPU is 4.

Eight 4090 servers fine-tune the 1b model, A single 4090 server fine-tunes the 170m model

**Comment by Henry-Ellis** [comment]
Thank you for your suggestions. The effect has improved a lot
